scripts/tagging/parseversion.py

```diff
@@ -52,18 +52,18 @@ def _parseVersion( self ):
       raise self.VersionStringError("Cannot parse this version string: %s" % self.version)
 
     if len(parts) >= 3:
-      if 'pre' == parts[2]:
+      if 'pre' == parts[2] or 'p' == parts[2]:
         self.pre = 0
-      elif 'pre' in parts[2]:
-        self.pre = int(parts[2].strip('pre'))
+      elif 'pre' in parts[2] or 'p' in parts[2]:
+        self.pre = int(parts[2].strip('pre').strip('p'))
```

Comment:
`strip('pre')` will strip all occurrences of the characters `p`, `r`, `e`, so no need to strip `p` again.

Reply:
Very true. Thanks